src/main/kotlin/no/nav/helse/spade/behandlinger/KafkaBehandlingerRepository.kt

```diff
@@ -54,4 +55,31 @@ class KafkaBehandlingerRepository(stream: BehandlingerStream) {
       log.error("unknown error while fetching state store", err)
       Either.Left(Feilårsak.UkjentFeil)
    }
+
+   fun getBehandlingerForPeriode(fom: String, tom: String): Either<Feilårsak, List<JsonNode>> = try {
+      stateStore.all().asSequence().filter { keyval ->
+         keyval.value.any {
+            it.has("avklarteVerdier") && it.path("avklarteVerdier").has("medlemsskap")
+               && it.path("avklarteVerdier").path("medlemsskap").has("vurderingstidspunkt")
+               && LocalDate.parse(it.path("avklarteVerdier").path("medlemsskap").get("vurderingstidspunkt").textValue().substring(0, 10)) >= (LocalDate.parse(fom))
```

Comment:
Et annet aspekt er om det egentlig trengs et "globalt" vurderingstidspunkt på søknaden, i tillegg til et for hvert enkelt vilkår. Men det er nok uansett out of scope :)

Reply:
Ja, det ligger allerede en oppgave på det i Trello 👍